proof_of_play.go

```diff
@@ -129,10 +129,22 @@ func (p proofOfPlay) publishEvent(name string, message string, level string) {
 	p.eventFn(name, message, "", level)
 }
 
-func (p *proofOfPlay) processResponse(popType string, adId string,
-	resp *http.Response) error {
-	code := resp.StatusCode
+func (p *proofOfPlay) makePoPRequest(req *http.Request,
+	popReq *PoPRequest, adId string, popType string) error {
+	resp, err := p.httpClient.Do(req)
+	if err != nil {
+		// Connection error - retry the request
+		p.processRequestFailure(popReq, err)
+		return &PoPError{
+			Status: http.StatusAccepted,
+			Message: fmt.Sprintf(
+				"Connection Error, Request will be Retried: %s", err.Error()),
```

Comment:
minor: it'd be better if we don't use mixed capital letters: Error / Request / Retried. Same on L177.

Reply:
Done